### DISCUSSION TITLE
何を探すか・探したいか

### DISCUSSION THREAD

**Discussion by ritsuxis** [ritsuxis/SoftwareEcosystem-Group1#3] [Ideas]
```
main, sub, placeholder, whitelist, denylistなど好ましい用語への置き換えについての議論。
```
をトピックにするとして、誰が何を探したいか、どういう分担にするか

**Comment by YSHRK** [comment]
変更したことで、人の流入・流出などの変化はあったのか

**Comment by ritsuxis** [comment]
BLM(ブラック・ライヴス・マター)の運動が元で色々変更がされていたりもしそう

単語としてBLMで探してみた時になにか見つかるのでは

https://ja.wikipedia.org/wiki/%E3%83%96%E3%83%A9%E3%83%83%E3%82%AF%E3%83%BB%E3%83%A9%E3%82%A4%E3%83%B4%E3%82%BA%E3%83%BB%E3%83%9E%E3%82%BF%E3%83%BC

**Comment by celorind** [comment]
好ましい用語(politically-correct euphemisms)が技術用語にも波及した経緯、次期

**Comment by YSHRK** [comment]
情報集めをしている人がいました
https://gist.github.com/mattn/488af4c3b3841ea901a1f9820636393c

**Reply by celorind** [reply]
[politically-correct-euphemisms](https://purlandtraining.com/2020/08/01/politically-correct-euphemisms/)
技術用語からははなれる例

**Reply by ritsuxis** [reply]
これを見た感じ、有名なリポジトリのPRを見れば探すことができそう

**Comment by ritsuxis** [comment]
単語として探すならmaster/slave, blacklist/whitelistかなあ

どの単語がPC扱いだとより言われているかも出せたら面白そう

**Comment by ritsuxis** [comment]
単語の変更を指摘するPRかissueに対する反対意見としてはどのようなものがあるか

**Comment by ritsuxis** [comment]
該当のPRを開けた人が普段からそのリポジトリにコミットしているのか、それとも単発なのか
